adapters/stroeerCore/stroeercore.go

```diff
@@ -0,0 +1,114 @@
+package stroeerCore
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type StroeerCoreAdapter struct {
+	Url string `json:"url"`
+}
+
+type StroeerRootResponse struct {
```

Comment:
I recommend the name `rootResponse` or simply `response`. There is no need to duplicate the package name and there is no need to export the type by starting with an uppercase letter.